src/components/Checkout/SocialShareV2.vue

```diff
@@ -168,11 +168,7 @@ export default {
 			});
 		},
 		linkedInShareUrl() {
-			return getFullUrl('https://www.linkedin.com/shareArticle', {
-				mini: 'true',
-				source: `https://${this.$appConfig.host}`,
-				summary: this.shareMessage.substring(0, 256),
-				title: `A loan for ${this.selectedLoan.name}`,
+			return getFullUrl('https://www.linkedin.com/sharing/share-offsite/', {
```

Comment:
The linked in sharing API has changed. It no longer supports any of these additional query params, only the url param of the page to be shared. The title and description are now pulled from the meta `og` tags.

Reply:
+1 Found out the same thing recently.